testing/ef_tests/src/cases/bls_batch_verify.rs

```diff
@@ -0,0 +1,72 @@
+use super::*;
+use crate::case_result::compare_result;
+use crate::impl_bls_load_case;
+use bls::{verify_signature_sets, BlsWrappedSignature, PublicKeyBytes, Signature, SignatureSet};
+use serde_derive::Deserialize;
+use std::borrow::Cow;
+use types::Hash256;
+#[derive(Debug, Clone, Deserialize)]
+pub struct BlsBatchVerifyInput {
+    pubkeys: Vec<PublicKeyBytes>,
+    messages: Vec<String>,
+    signatures: Vec<String>,
+}
+
+#[derive(Debug, Clone, Deserialize)]
+pub struct BlsBatchVerify {
+    pub input: BlsBatchVerifyInput,
+    pub output: bool,
+}
+
+impl_bls_load_case!(BlsBatchVerify);
+
+impl Case for BlsBatchVerify {
+    fn result(&self, _case_index: usize, _fork_name: ForkName) -> Result<(), Error> {
+        let messages = self
+            .input
+            .messages
+            .iter()
+            .map(|message| {
+                hex::decode(&message[2..])
+                    .map_err(|e| Error::FailedToParseTest(format!("{:?}", e)))
+                    .map(|bytes| Hash256::from_slice(&bytes))
+            })
```

Comment:
```suggestion
            .map(|s| Hash256::from_str(s).map_err(|e| Error::FailedToParseTest(format!("{:?}", e))))
```

Just a minor simplification.